lib/block_view_profile_test.go

```diff
@@ -794,6 +795,39 @@ func TestUpdateProfile(t *testing.T) {
 			1.5*100*100,                  /*newStakeMultipleBasisPoints*/
 			false /*isHidden*/)
 	}
+	// Should not allow ENS names before block height, should allow after
+	{
+
+		params.ForkHeights.UnlimitedDerivedKeysBlockHeight = uint32(100)
+		_, _, _, err = _updateProfile(
+			t, chain, db, params,
+			1,                            /*feeRateNanosPerKB*/
+			m3Pub,                        /*updaterPkBase58Check*/
+			m3Priv,                       /*updaterPrivBase58Check*/
+			m5PkBytes,                    /*profilePubKey*/
+			"m5.eth",                     /*newUsername*/
+			"m5 created by paramUpdater", /*newDescription*/
+			otherShortPic,                /*newProfilePic*/
+			11*100,                       /*newCreatorBasisPoints*/
+			1.5*100*100,                  /*newStakeMultipleBasisPoints*/
+			false /*isHidden*/)
+
+		require.Error(err)
+		require.Contains(err.Error(), RuleErrorInvalidUsername)
+
+		params.ForkHeights.UnlimitedDerivedKeysBlockHeight = uint32(0)
```

Comment:
instead of modifying the fork height, please mine more blocks so we go past the fork height.

Reply:
Resolving due to miner sync issues